modules/post/windows/gather/forensics/fanny_bmp_check.rb

```
@@ -0,0 +1,84 @@
+                keys = query.split(/,/)
+                begin 
+                        keys.each do |key|
+                                (key, value) = parse_query(key)
+                                has_key = registry_enumkeys(key)
+                                has_val = registry_enumvals(key)
+
+                                if has_key.include?(value) or has_val.include?(value)
+                                        print_good("#{sysinfo['Computer']}: #{key}\\#{value} found in registry.")                                                                                                                                                                                               
+                                        match += 1                                                                                                                                                                                                                                                              
+                                end                                                                                                                                                                                                                                                                             
+                        end                                                                                                                                                                                                                                                                                     
+                rescue; end                                                                                                                                                                                                                                                                                     
```

Don't silently swallow exceptions.